docs/internal/investigations/runtime-governor-poc/README.md

```diff
@@ -0,0 +1,138 @@
+
+![Architecture Diagram after reconfiguration](./stage-2.svg)
+
+Tools:
+- [Docker](https://www.docker.com/)
+- [k3d v3](https://github.com/rancher/k3d)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Helm 3](https://helm.sh/)
+- watch (`brew install watch`)
+
+Make sure that you have all the tools installed, and start in the `poc` directory.
+
+> **NOTE**: Keep in mind that some shells automatically escape some characters when copying and pasting commands. Always check the command before running it.
+
+### Control Plane deployment
+
+In the first step, we deploy the Control Plane cluster which hosts the Runtime Governor component as well as two Redis instances for the purpose of persisting the state by the Runtimes.
+
+To deploy the Control Plane cluster, run the following command:
+
+```bash
+./setup-cp.sh
+```
+
+You can watch the installation progress by looking at the statuses of the Pods that are deployed in the cluster:
+
+```bash
+watch kubectl --context=k3d-cp get po --all-namespaces
+```
+
+### Runtimes deployment
+
+The next step is to deploy two Runtimes that will consume the Control Plane API and persist the state in the Redis instances configured by the central Control Plane.
+
+To deploy the first Runtime cluster, run the following command. It will create the cluster and deploy a basic set of components, including the NodeApp that demos the use case with the state store.
+
+```bash
+./setup-runtime1.sh
+```
+
+You can watch the installation progress by looking at the statuses of Pods that are deployed in the cluster. At this stage, there should be a NodeApp Pod deployed, but the Dapr sidecar should not be injected yet.
+
+```bash
+watch kubectl --context=k3d-runtime1 get po --all-namespaces
+```
+
+Next, deploy the Runtime Agent component using the following command. It will deploy the Runtime Agent that connects to the Control Plane cluster and fetches the configuration for the given Runtime.
+
+```bash
+./setup-runtime1-agent.sh
+```
+
+You can watch the installation progress by looking at the statuses of Pods that are deployed in the cluster. After a successful deployment of the Runtime Agent component, it should fetch the configuration and apply it to the cluster. The NodeApp Pod should be recreated with the Dapr sidecar injected.
+
+```bash
+watch kubectl --context=k3d-runtime1 get po --all-namespaces
+```
+
+In order to run the second Runtime cluster, please run all the previous steps using the `runtime2` scripts.
+
+### Use case scenario
+
+The setup described gives you the possibility to interact with two Runtime clusters. On both Runtimes, there is an example Dapr NodeApp deployed. It exposes two endpoints, called `/order` and `/neworder`, which interact with the Redis state store. You can call those endpoints using a Docker container running in the same Docker network as the clusters. 
```

Comment:
```suggestion
The setup described gives you the possibility to interact with two Runtimes. 
On both Runtimes, there is an example Dapr NodeApp deployed. 
It exposes two endpoints, called `/order` and `/neworder`, which interact with the Redis state store. 
You can call those endpoints using a Docker container running in the same Docker network as the clusters. 
```

1) Should we also change `Runtime clusters` to `Runtimes` here? 
2) I added the new lines for the ease of editing. The sentences will still render the same.